reposcan/reposcan.py

```diff
@@ -37,7 +37,7 @@
 def init_db():
     """Setup DB connection parameters"""
     DatabaseHandler.db_name = os.getenv('POSTGRESQL_DATABASE', "vmaas")
-    DatabaseHandler.db_user = os.getenv('POSTGRESQL_USER', "vmaas_user")
+    DatabaseHandler.db_user = os.getenv('POSTGRESQL_USER', "vmaas_writer")
     DatabaseHandler.db_pass = os.getenv('POSTGRESQL_PASSWORD', "vmaas_passwd")
```

Comment:
Please change the default password for the vmaas_writer.

Reply:
Having code default the user if it isn't set in the env is one thing - but the passwd shouldn't be hard-coded. That's why I left that as I did.

Reply:
It's only a default password (overridden by Dockerfile value and overridden by ENV map file value), in `webapp/database.py` you are changing the default pw too.